docs/source/whatsnew.rst

```diff
@@ -1,17 +1,21 @@
-What's new in the v2.0 Beta release
-===================================
+What's new in Hyperledger Fabric v2.0
+=====================================
 
-The Beta release of Hyperledger Fabric v2.0 allows users to try out upcoming
-features of Fabric v2.0. While the v2.0 release is not yet production ready,
-the Beta is feature complete, meaning it includes all the features that are
-expected to be delivered in the final v2.0 release.
+As the first Hyperledger Fabric major release since v1.0, Fabric v2.0
+delivers important new features and changes to users and operators alike,
```

Comment:
```suggestion
delivers important new features and changes for users and operators alike,
```
Nit: I think this reads better as "for" instead of "to" here.

Reply:
changing.